vmm/src/config.rs

```diff
@@ -928,6 +929,11 @@ impl DiskConfig {
             .map_err(Error::ParseDisk)?
             .unwrap_or(Toggle(false))
             .0;
+        let disable_aio = parser
```

Comment:
We don't really need this - we only added `_disable_io_uring` as we supported kernels that didn't have it and we needed some way to test. Our minimum kernel now does support io_uring so we can remove this at some point.